vonk/reference/programming_api/bundles.rst

```diff
@@ -0,0 +1,145 @@
+.. _vonk_reference_api_bundles:
+
+Constructing bundles
+====================
+
+In a Plugin or Facade you may need to construct a bundle from a set of resources, e.g. a SearchResult (see :ref:`here <vonk_reference_api_isearchrepository>`). There are two ways of doing this: with the Bundle POCO or with SourceNodes.
+
+Bundle POCO
+-----------
+
+This is fairly straightforward. Create a new Bundle object, and fill its properties, iterating over the set of resources that you have. The code looks like this:
+
+.. code-block:: csharp
+
+   using Hl7.Fhir.Model; //Either from Hl7.Fhir.Core.Stu3 or Hl7.Fhir.Core.R4
+
+   ...
+
+      var searchResult = await searchRepository.Search(args, options)
+      var bundle = new ResourceKey(resourceType, resourceId);
```

Comment:
For what do we need the ResourceKey? Bundle needs also to be initialised with new Bundle.
I would also add a comment that Bundle.type is mandatory.